app/jobs/process_github_sponsor_update_job.rb

```diff
@@ -1,12 +1,19 @@
 class ProcessGithubSponsorUpdateJob < ApplicationJob
   queue_as :default
 
-  def perform(action, gh_username)
-    return unless action == 'created'
-
+  def perform(action, gh_username, node_id, privacy_level, is_one_time, monthly_price_in_cents)
     user = User.find_by(github_username: gh_username)
```

Comment:
Do we want to attempt to automatically update a user's github_username if we can't find it? We try to etch the user's email via another API call (it is not in the payload I think)